tdvt/tdvt/config_gen/gentests.py

```diff
@@ -2,20 +2,21 @@
     Generate datasource specific logical query files based on a set of genericized input files.
 """
 
-import sys
+import glob
+import logging
 import os
+import re
 import shutil
-import argparse
-from .templates import *
 from string import Template
-import re
-import glob
-from ..resources import *
```

Comment:
No TDVT code was being imported and used; seems like the only import was for `logging`, which I added on line 6.